data/human/news.txt

```diff
@@ -2392,3 +2393,60 @@ news "graffiti on spaceport wall"
 			"Because I could not stop for Death - He kindly stopped for me - The Shuttle held but just one bunk - Most Peacefully. / We slowly flew; He knew no haste - And I had put away - My labor and my leisure too, - For His Civility." # Based on Emily Dickinson's "Because I could not stop for Death"
 			"We passed the Carrier, where Soldiers strove - In Fighters; in the Void - We passed the Bays with Docked Drones - We passed the Hardened Hull. / Or rather; It passed Us - The Dew drew quivering and Chill - For decompressed, my spacesuit - My tether, torn away." # Based on Emily Dickinson's "Because I could not stop for Death"
 			"We paused before a House that seemed - A Cylinder to be launched - The Roof was scarcely visible - The Flag; covering around. / Since then, 'tis Centuries, and yet - Feels shorter than the Day - I first surmised the Torpedo Bay - Aimed toward Eternity." # Based on Emily Dickinson's "Because I could not stop for Death"
+
+news "ccor graffiti"
+	location
+		planet "Gagarin"
+	name
+		word
+			"Graffiti painted on a wall"
+	message
+		word
+			"Someone has stenciled the emblem of the Confederated Councils of the Outer Rim here. ${ccor crest graffiti}"
+			"A crest of the local government here has been defaced with a generous amount of spray paint. ${ccor crest graffiti}"
+			"The insignia of a local guerrilla group has been painted on the wall here. ${ccor crest graffiti}"
+			"There's a spray-painted mess of competing emblems and slogans here. You can't make out the details, but it appears as if the burgundy-colored markings are in support of the CCOR."
+
+phrase "ccor crest graffiti"
+	word
+		`` 2
+		`Underneath are the words "BELENOS SHINES ACROSS THE GALAXY."`
+		`Underneath are the words "THE CONFEDERATED COUNCILS WILL RETURN."`
+		`Underneath are the words "RESIST THE FOREIGN OCCUPIERS."`
+		`Underneath are the words "THE CCOR WILL RISE AGAIN."`
+		`Underneath are the words "LIBERATE THE OUTER RIM."`
+		`Underneath are the words "DISMANTLE THE SYNDICATE OLIGARCHY."`
```

Comment:
Can we not repeat "Underneath are the words " six times here?

Reply:
I meant "can we not write it out so many times" not "can we have different options".